packages/sdk/test/evm/gasless.test.ts

```diff
@@ -47,4 +48,43 @@ describe("Gasless Forwarder", async () => {
 
     await pack.open(0, 0);
   });
+
+  it.skip("should mint batch via multicall tx with relayer", async () => {
+    const NFT_COLLECTION_ADDRESS = "0x24A5aB2878B63716B001aa9AbE816c2662192B12";
+    const provider = ethers.getDefaultProvider(RPC_URL);
+    // const wallet = Wallet.createRandom().connect(provider);
+    const wallet = new Wallet("<PRIVATE KEY>", provider);
```

Comment:
since ur doing gasless here u could just generate a fresh wallet via `ethers.wallet.createRandom()`? right (the wallet doesn't need any funds)

Reply:
Had to test with admin controlled functions, so needed the deployer's key. Removed the key after testing and marked the test as `skip`.